tests/manager/manager-ctor_error-003.phpt

```diff
@@ -28,4 +32,6 @@ OK: Got MongoDB\Driver\Exception\InvalidArgumentException
 Unsupported readPreference value: 'nothing'
 OK: Got MongoDB\Driver\Exception\InvalidArgumentException
 Primary read preference mode conflicts with tags
+OK: Got MongoDB\Driver\Exception\InvalidArgumentException
+Read preference tags must be an array of zero or more documents
```

Comment:
This is more consistent with the ReadPreference constructor and I think it's more helpful to the user. We're validating individual values before we check the combined state of the read preference.

On that note, I think there's an oversight in `php_phongo_apply_rp_options_to_uri()` where we're never checking if "maxstalenessms" is a positive value. As-is, I think a negative value would only trigger a generic "Read preference is not valid" error.